src/app/Service/Data/Data.service.ts

```diff
@@ -0,0 +1,64 @@
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { MusicService } from '../Music/music.service';
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class datatService {
```

Comment:
```suggestion
export class DataService {
```
class ai lại viết thường chữ đầu, với viết sai ngữ pháp. 
Ở vscode tại extension để kiểm tra lỗi chính tả nhé, như bên dưới
![image](https://github.com/VanHuy-2904/Clone-Spotify/assets/51251932/26b4bc10-e5b4-45af-8be7-5d1aa96b71cd)

Reply:
@tdmtri99  dạ em sửa r ạ